fe/endpoint_correction.py

```diff
@@ -18,12 +18,11 @@ def exp_u(rotation, k, beta):
 def sample_multiple_rotations(k, beta, size):
     num_batches = 500
     batch_size = 10000
-
+    state = np.random.RandomState(2021)
```

Comment:
Will we need to update this on Jan 1st? :stuck_out_tongue:

Reply:
We generally have the convention of setting the random seed as the year we made the change, doesn't really matter as long as the function is deterministic